keep-notes/src/context/auth-context.jsx

```diff
@@ -0,0 +1,63 @@
+import { createContext, useContext, useState } from "react";
+import axios from "axios"
+
+const AuthContext = createContext();
+
+const AuthProvider = ({children})=>{
+    const localStorageToken = JSON.parse(localStorage.getItem("loginCred"))
+    console.log(localStorageToken);
```

Comment:
Remove console

Reply:
Hi Manish,
I have removed irrelevant console log, added barrel import and used console.error() whereever required.
Please review it.